framework/set/setRKE1Config.go

```diff
@@ -83,7 +63,38 @@ func SetRKE1(clusterName, k8sVersion, psact string, nodePools []config.Nodepool,
 
 	rootBody.AppendNewline()
 
-	clusterSyncNodePoolIDs := ``
+	servicesBlock := rkeConfigBlockBody.AppendNewBlock("services", nil)
+	servicesBlockBody := servicesBlock.Body()
+
+	etcdBlock := servicesBlockBody.AppendNewBlock("etcd", nil)
+	etcdBlockBody := etcdBlock.Body()
+
+	backupConfigBlock := etcdBlockBody.AppendNewBlock("backup_config", nil)
+	backupConfigBlockBody := backupConfigBlock.Body()
+
+	backupConfigBlockBody.SetAttributeValue("enabled", cty.BoolVal(true))
+	backupConfigBlockBody.SetAttributeValue("interval_hours", cty.NumberIntVal(12))
+	backupConfigBlockBody.SetAttributeValue("safe_timestamp", cty.BoolVal(true))
+	backupConfigBlockBody.SetAttributeValue("timeout", cty.NumberIntVal(120))
+
```

Comment:
would it be worth setting default consts for these hard coded ints and strings? are these just what we have by default in rancher, and you have to manually set it in tf else they are set to nil?

Reply:
This is an oversight, I actually meant to have these be dynamically set. I'll change that.

Reply:
Resolved